rnaget-openapi.yaml

```diff
@@ -587,6 +673,14 @@ definitions:
     externalDocs:
       description: "Find out more"
       url: "https://github.com/ga4gh-rnaseq/schema"
+  error:
+    type: "object"
+    description: "General API error model"
+    properties:
+      code:
```

Comment:
Do we need this as a property? The response code should be available directly in the response object.

Reply:
I don't think it's a required property. I added it as it seemed convenient to me.
I can remove it if you prefer to have a simpler error model.

Reply:
Any though on this?

Reply:
I think we should keep things as simple as possible - we can add fields later if a use case arises

Reply:
Sounds good. I just removed the `code` property.